src/index.ts

```diff
@@ -120,6 +121,7 @@ export default async function GleeAppInitializer () {
   })
 
   app.on('adapter:server:ready', (e: EnrichedEvent) => {
+    generateDocs(parsedAsyncAPI, config, null)
```

Comment:
Why here? If we generate docs on "adapter:server:ready", it will only work for WebSocket and HTTP servers but not for clients (MQTT, Kafka, etc.). I think it should be generated as soon as possible and not necessarily when an event happens.